scripts/create-install-iso.sh

```diff
@@ -268,18 +268,22 @@ else
     mkdir "$SCRATCHDIR/grub"
     rpm2cpio $SCRATCHDIR/grub-efi-*.rpm | (cd "$SCRATCHDIR/grub" && cpio ${VERBOSE} -idm)
 
-    BOOTX64=$(mktemp "$TMPDIR/bootx64-XXXXXX.efi")
-
-    "$MKIMAGE" --directory "$SCRATCHDIR/grub/usr/lib/grub/x86_64-efi" --prefix '()/EFI/xenserver' \
-               $VERBOSE \
-               --output "$BOOTX64" \
-               --format 'x86_64-efi' --compression 'auto' \
-               'part_gpt' 'part_msdos' 'part_apple' 'iso9660'
-
-    # grub modules
-    # FIXME: too many modules?
-    tar -C "$SCRATCHDIR/grub/usr/lib" -cf - grub/x86_64-efi |
-        tar -C "$ISODIR/boot" -xf - ${VERBOSE}
+    if false; then
```

Comment:
Either we drop it, or add a comment explaining what this skipped part of the script use to do?